internal/env/config.go

```diff
@@ -97,7 +103,16 @@ func NewConfig(stdout, stderr io.Writer) (*Config, error) {
 		cfg.logDebug()
 	}
 
-	cfg.deprecationWarn()
+	// Deprecation messages
+	cfg.varsMessage(deprecatedVars, "Warning: ignoring deprecated '%s' setting\n")
+
+	if cfg.Command != "convert" {
+		cfg.varsMessage(convertVars, "Warning: ignoring '%s' setting as is only used when 'mode' is 'convert'\n")
+	}
```

Comment:
This is redundant, the block below serves the exact same purpose.

Reply:
thx, leftover from my previous tests